.github/workflows/tests.yml

```diff
@@ -56,6 +56,15 @@ jobs:
           unzip -o dcm2niix_lnx.zip
           sudo install dcm2nii* /usr/bin/
 
+      - name: SpinalCord Toolbox installation
+        run: |
+          cd ../..
+          git clone --depth 1 -b 5.0.1 https://github.com/neuropoly/spinalcordtoolbox
```

Comment:
Ah, sorry, the fix for https://github.com/neuropoly/spinalcordtoolbox/issues/3112 isn't released yet. You'll have to use `-b master` here. I guess we'll try to get a 5.0.2 out that has the fix in in a day or two.

Reply:
That would be in line with what Julien suggested doing already: https://github.com/shimming-toolbox/shimming-toolbox/pull/184#discussion_r536935477, so win-win.